frontend/integration-tests-cypress/cypress.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
We should consider where to put tests now that we have a monorepo structure.

cc @christianvogt @vojtechszocs

Reply:
IMO, each component should manage their own tests.
Perhaps for cypress, we start with the tests in packages. 
(@dtaylor113 that could mean moving this set of monitoring tests into a `frontend/packages/...` dir, even if the monitoring components are still in `frontend/public/monitoring/`. We can move the code later). 

The problem with that might be sharing helpers, etc.  But if helpers live in `frontend/packages/shared/cypress`, components are welcome to import the ones they want.

Reply:
Ok with "We can move the code later [to 'frontend/packages/shared/cypress']".  I'd like to understand the package structure a little better before making any changes.

Reply:
Moved to `frontend/packages/integration-tests-cypress`